src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,63 @@
+$cell-side: 100px;
+$cell-sideAndGap: $cell-side + 1;
+$weekDays: (
+  "mon": 0,
+  "tue": $cell-sideAndGap * 1,
+  "wed": $cell-sideAndGap * 2,
+  "thu": $cell-sideAndGap * 3,
+  "fri": $cell-sideAndGap * 4,
+  "sat": $cell-sideAndGap * 5,
+  "sun": $cell-sideAndGap * 6
+);
+
+.calendar {
+  &__list {
+    display: flex;
+    gap: 1px;
+    padding: 10px;
+    margin: 0;
+    box-sizing: border-box;
+    list-style: none;
+    flex-wrap: wrap;
+    width: $cell-side * 7 + 20px + 6px;
+
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    width: 100px;
+    height: 100px;
+    border: solid 1px #000;
+    box-sizing: border-box;
+    transition: transform 0.5s;
```

Comment:
As I see in hover you change not only a transform property, let's add a transition to background-color too

Reply:
Thanks for your review, have already done all changes)